### PR TITLE
removing SeccompProfilePath

### LINKED ISSUE #117045: Remove SeccompProfilePath
Look for "// TODO: Deprecated, remove after we switch to Seccomp field". Fix it. See https://github.com/kubernetes/kubernetes/pull/115898 as example.

/cc @saschagrunert 

/help
/good-first-issue
/sig node

add @saschagrunert as a reviewer when sending a PR.

(issue is a follow up from https://github.com/kubernetes/kubernetes/pull/117020/files#r1154025906)

### PR DESCRIPTION
**Pull request by sherifhmdy**
removing SeccompProfilePath after being replaced by  Seccomp

#### What type of PR is this?
/help
/good-first-issue
/sig node

#### What this PR does / why we need it:
removing SeccompProfilePath. Because it's been replaced by Seccomp
#### Which issue(s) this PR fixes:
fixing issue 

Fixes #117045

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: pkg/kubelet/kuberuntime/security_context.go [hunk 1]
```diff
@@ -35,14 +35,7 @@ func (m *kubeGenericRuntimeManager) determineEffectiveSecurityContext(pod *v1.Po
 		}
 	}
 
-	// TODO: Deprecated, remove after we switch to Seccomp field
-	// set SeccompProfilePath.
```
Comment (wzshiming) [inline pkg/kubelet/kuberuntime/security_context.go:39]: `getSeccompProfilePath` the method is also to be removed

Comment (sherifhmdy) [reply pkg/kubelet/kuberuntime/security_context.go:39]: already removed

Comment (sherifhmdy) [reply pkg/kubelet/kuberuntime/security_context.go:39]: done

**Review by saschagrunert — CHANGES_REQUESTED** [review]
The linter is unhappy with this:

```
pkg/kubelet/kuberuntime/helpers_linux_test.go:416:28: m.getSeccompProfilePath undefined (type *kubeGenericRuntimeManager has no field or method getSeccompProfilePath)
pkg/kubelet/kuberuntime/helpers_linux_test.go:497:28: m.getSeccompProfilePath undefined (type *kubeGenericRuntimeManager has no field or method getSeccompProfilePath) (typecheck)
```

**Review by sherifhmdy — COMMENTED** [review]
I've removed the test related to getSeccompProfilePath from helpers_linux_test.go

**Pull request closed without merging**